packages/react/.storybook/preview.js

```diff
@@ -15,25 +12,25 @@ configureActions({
   depth: 3,
 });
 
-addParameters({
+export const parameters = {
```

Comment:
addParameters/addDecorators are deprecated and instead exported direct from the preview file.